src/aura/PreviewYourInvoiceCmp/PreviewYourInvoiceCmp.cmp

```diff
@@ -0,0 +1,21 @@
+<aura:component controller="InvConfirmation" implements="force:appHostable,flexipage:availableForAllPageTypes,force:hasRecordId" access="global">
+    <aura:attribute name="invoiceId" type="String"/>
+    <aura:attribute name="accountId" type="String"/>
+    <aura:attribute name="sellerId" type="String"/>
+    <aura:attribute name="serviceId" type="Set"/>
+    <lightning:notificationsLibrary aura:id="notifLib"/>
+    <div class="slds-card slds-p-top_medium slds-p-around_small" >
+        <div class= "slds-grid slds-wrap">
+            <iframe src="{!'https://ocinvoice-dev-ed.lightning.force.com/apex/Invoice?invId=' + v.invoiceId}" width="100%" height="500px;" frameBorder="0"/>
```

Comment:
Zapomnaiłem o tym, ten link trzeba budować dynamicznie bo na innym środowisku nam nie zadziała. Trzeba budować to po stronei apex'a neistety(jest funkcja ale tylko w visualforce). Tutaj przykład:

https://salesforce.stackexchange.com/questions/174781/what-is-the-lightning-component-equivalent-for-site-baseurl

Reply:
Dodam że potrzebujemy tylko funkcji _getBaseUrl()_